pkg/network/http/telemetry.go

```diff
@@ -80,39 +73,32 @@ func (t *telemetry) aggregate(txs []httpTX, err error) {
 func (t *telemetry) reset() telemetry {
 	now := time.Now().Unix()
 	then := t.then.Swap(now)
+	elapsed := now - then
+
+	delta := map[string]int64{}
```

Comment:
You will have to make this a `map[string]interface{}` and then case the values below to `int64`.

Reply:
Actually the return at the end does not work: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/153017500

Reply:
Ah, thanks -- I'll get that fixed up (maybe by not returning `telemetry`)